### ISSUE TITLE
Swift 2.3

### ISSUE THREAD

**Issue by ProudOfZiggy** [FolioReader/FolioReaderKit#143]
How I can get FolioReaderKit for Swift 2.3? 
Your dependency - AEXML automatically using swift 3.....
RealmSwift also have troubles with swift 2.3 https://github.com/realm/realm-cocoa/issues/3894
I really need this...

**Comment by ProudOfZiggy** [comment]
Actually I don't understand how you are using swift 3.0 module while FolioReader is still using swift 2.3

**Comment by hebertialmeida** [comment]
FolioReader uses a compatible version from AEXML, I is on schedule the migration.

**Comment by ProudOfZiggy** [comment]
Well, Not works for me, It saying that Folio trying to use swift 3.0 module while Folio is on 2.3

**Comment by hebertialmeida** [comment]
Fixed that on version 1.9.1, the problem was that the versions was not specified, sometimes it was using the most recent version.

**Comment by hebertialmeida** [comment]
There is a branch for Swift 2.3:
```ruby
pod 'FolioReaderKit' :git => 'https://github.com/FolioReader/FolioReaderKit.git', :branch => 'swift23'
```
